peer/brontide.go

```diff
@@ -4153,6 +4160,46 @@ func (p *Brontide) handleRemovePendingChannel(req *newChannelMsg) {
 	p.addedChannels.Delete(chanID)
 }
 
+// handlePeerStorageMessage handles `PeerStorage` message, it stores the message
+// and sends it back to the peer as an ack.
+func (p *Brontide) handlePeerStorageMessage(msg *lnwire.PeerStorage) error {
+	var sendMsg lnwire.Message
+
+	// Check if we have the feature to store peer backup enabled.
+	if p.cfg.PeerDataStore != nil {
+		// Store the data if there are active channels, if not,
+		// do nothing.
+		if p.hasActiveChannels() {
+			// Store the data.
+			err := p.cfg.PeerDataStore.Store(msg.Blob)
+			if err != nil {
+				return err
+			}
+
+			// Send back message as ack.
+			sendMsg = lnwire.NewPeerStorageRetrievalMsg(msg.Blob)
```

Comment:
the spec no longer says anything about sending back an ack. This is now only done on connection re-establishment